catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/PaimonCatalogPropertiesMetadata.java

```diff
@@ -61,6 +62,12 @@ public class PaimonCatalogPropertiesMetadata extends BaseCatalogPropertiesMetada
           AuthenticationConfig.AUTH_TYPE_KEY,
           AuthenticationConfig.AUTH_TYPE_KEY);
 
+  private static final Map<String, String> S3_CONFIGURATION =
+      ImmutableMap.of(
+          PaimonS3FileSystemConfig.S3_ACCESS_KEY, PaimonS3FileSystemConfig.S3_ACCESS_KEY,
```

Comment:
I prefer to use the properties defined in `S3Properties` to unify the storage configuration for Gravitino

Reply:
Please see https://github.com/apache/gravitino/pull/4939#discussion_r1764880225